src/main/shell-sync.ts

```diff
@@ -22,23 +22,16 @@
 import { shellEnv } from "./utils/shell-env";
 import os from "os";
 import { app } from "electron";
-
-interface Env {
-  [key: string]: string;
-}
+import logger from "./logger";
+import { isSnap } from "../common/vars";
 
 /**
  * shellSync loads what would have been the environment if this application was
  * run from the command line, into the process.env object. This is especially
  * useful on macos where this always needs to be done.
  */
 export async function shellSync() {
-  const { shell } = os.userInfo();
-  let envVars = {};
-
-  envVars = await shellEnv(shell);
-
-  const env: Env = JSON.parse(JSON.stringify(envVars));
+  const env = await shellEnv(os.userInfo().shell);
```

Comment:
Just wondering why we are passing always users default shell and not the current one (if current is set). Probably story for another issue/PR.

Reply:
Yeah probably for a different PR. Don't think this is wrong though, where else would they define it?